### DISCUSSION TITLE
Creating page with a title that has a colon creates two pages :o

### DISCUSSION THREAD

**Discussion by ThatAdamGuy** [athensresearch/athens#1163] [Archive]
Reproducible.

Steps:
1) Create page with a colon in it, e.g., "Day1: Monday"
2) Witness that you now have two pages:  "Day1" and "Day1: Monday"

**Comment by jefftangx** [comment]
Whoa that is really weird! Probably something to do with parser.

**Comment by datokrat** [comment]
How do you create the page "Day1: Monday"? When I create it using Athena, my Athens does not create a an additional page called "Day1". I'm using Linux and Athens v1.0.0-BETA.80. I also tried the [web version](https://athensresearch.github.io/athens).

(Not sure whether I should post this here or in the issue...)

See [this video](https://www.loom.com/share/8a1c5ae1f4b9468eb6c3e49382a6a1ba).

**Reply by jefftangx** [reply]
I write [[Day 1: Monday]] on a block, not from Athena.

**Reply by datokrat** [reply]
Strange. I tried that too, right now in the web version, and it seems to work fine: [Loom](https://www.loom.com/share/82b660580d09440592ef14d0c98d157d)
